Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM python:3.7.3
+LABEL maintainer="cgphelp@sanger.ac.uk" \
+      uk.ac.sanger.cgp="Cancer, Ageing and Somatic Mutation, Wellcome Sanger Institute" \
+      version="0.1.0" \
+      description="SigProfiler docker"
+ENV NEWUSER=cgp \
+    HOME=/opt/cgp
+WORKDIR $HOME/build_dir
+COPY . .
+RUN useradd --shell /bin/bash -g 0 $NEWUSER \
```

Comment:
This `RUN` combines too many unrelated items:

* User creation should be separate, normally after the install process
* grouping apt related items is sensible (inc the sources.list stuff)
* separate the install of the repo code to another `RUN`